modules/db/test/blaze/db/impl/search_param/near/geo_spec.clj

```diff
@@ -0,0 +1,8 @@
+(ns blaze.db.impl.search-param.near.geo-spec
+  (:require
+   [blaze.db.impl.search-param.near.geo.spec :as gs]
+   [clojure.spec.alpha :as s]))
+
+(s/fdef haversine-distance
+  :args (s/cat :loc-1 ::gs/coordinates :loc-2 ::gs/coordinates)
```

Comment:
the specs should have the namespace `blaze.db.impl.search-param.near.geo`. I never have the `spec` as part of a data spec namespace. please use `:as-alias` at `require` if you don't need to import the geo ns. also the alias could be called `geo`.